src/app/OIDCClients/data/index.js

```diff
@@ -0,0 +1,27 @@
+const mapEntity = async (entity) => {
+  if (!entity) {
+    return undefined;
+  }
+  return {
+    friendlyName: entity.name,
+    client_id: entity.relyingParty.client_id,
+    client_secret: entity.relyingParty.client_secret,
+    api_secret: entity.relyingParty.api_secret || undefined,
+    redirect_uris: entity.relyingParty.redirect_uris,
+    post_logout_redirect_uris: entity.relyingParty.post_logout_redirect_uris,
+    params: entity.relyingParty.params,
+  };
```

Comment:
Missing mappsings for:
1. token_endpoint_auth_method
2. service_home
3. postResetUrl
4. grant_types
5. response_types